### PR TITLE
Revert "wait_until_not_moving should consider 'running holding' as 'n…

### PR DESCRIPTION
**Pull request by dwalton76**

### CODE REVIEW

**Comment by dwalton76** [comment]
Resolves #605

File: debian/changelog [hunk 1]
```diff
@@ -11,7 +11,6 @@ python-ev3dev2 (2.0.0~beta3) stable; urgency=medium
   * Sound: removed play() in favor of play_file()
   * suppress unittest ResourceWarning messages
   * Added DistanceValue classes
-  * wait_until_not_moving should consider "running holding" as "not moving"
```
Comment (WasabiFan) [inline debian/changelog]: We shouldnt remove things from the released changelog...

**Review by WasabiFan — APPROVED** [review]
Should add a new changelog entry

**Comment by dwalton76** [comment]
> Should add a new changelog entry

what should the new header be in the changelog?  Or do we keep the current one?

**Comment by WasabiFan** [comment]
Create a new one and replace "stable" with "UNRELEASED" for now.

**Review by WasabiFan — APPROVED** [review]

**Pull request merged**